packages/perseus/src/widgets/radio/__stories__/indicator.stories.tsx

```diff
@@ -0,0 +1,153 @@
+import {HeadingMedium} from "@khanacademy/wonder-blocks-typography";
+import * as React from "react";
+
+import Indicator from "../choice-indicator.new";
+
+const Container = (props: {children: React.ReactNode}): React.ReactElement => {
+    return (
+        <div
+            style={{
+                padding: "10px",
+                display: "flex",
+                flexDirection: "column",
+                gap: "10px",
+                marginBlockEnd: "25px",
+            }}
+        >
+            {props.children}
+        </div>
+    );
+};
+
+export default {
+    title: "Perseus/Widgets/Radio/Indcator",
```

Comment:
No need to change in this PR. Just a heads up on our proposal for the new documentation any internal components to the widget will be under
```suggestion
    title: "Perseus/Widgets/Radio/Widget Internal Components/Indcator",
```